### PR TITLE
Simplify the front page

### PR DESCRIPTION
**Pull request by jimwins**
Just a first (deep) cut to not be so backward-looking and re-focus on how contributors can get involved.

### CODE REVIEW

File: www/index.php [hunk 1]
```diff
@@ -5,34 +5,27 @@
...
-some improvements from the others on an occasional basis (the PEAR
-documentation team uses a variant of the revcheck developed for phpdoc for
-example). This site is aimed at getting documentation efforts closer, and
-provide more tools and information for manual authors and translators.</p>
+
+<p>
+  This site has instructions and tools for building, translating, and
+  maintaining the 
+  <a href="https://www.php.net/docs.php">PHP Documentation</a>, which
+  is currently availabe in nine languages plus the original English.
```
Comment (saundefined) [inline www/index.php:13]: IMO it's clearer, thanks!

nit: I wouldn't specify the exact number, there are a bit more translations (http://doc.php.net/revcheck.php), they are just not displayed on the main site. I'm afraid if some translations become active, or conversely abandoned, we'll forget to update the value here :)

Comment (jimwins) [reply www/index.php]: Updated this to just say 'multiple'.

**Review by saundefined — APPROVED** [review]
Looks good, thanks!

If there's no objection, I'll merge it later today.

**Comment by lhsazevedo** [comment]
Looks really good

**Pull request merged**